_includes/footer.html

```diff
@@ -0,0 +1,56 @@
+<footer class="c-footer">
+  <div class="c-footer__container">
+    <section class="c-footer__award-container">
+      <img src="../assets/Group 2.svg" class="c-footer__award-image" alt="">
+      <p class="c-footer__award-text">named one of the best workplaces to work in ohio for 2016, 2017 and 2018</p>
+    </section>
+    <div class="c-footer__link-lists">
```

Comment:
I was a little confused by this naming at first, maybe it could be something like `c-footer__section` or `c-footer__column` to be a little more descriptive 🙂

Reply:
makes sense :)